SilKit/source/config/ParticipantConfiguration_Full.json

```
@@ -232,7 +232,8 @@
   },
   "Experimental": {
     "TimeSynchronization": {
-      "AnimationFactor": 1.5
+      "AnimationFactor": 1.5,
+      "EnableMessageAggregation": "Auto" 
```

```suggestion
      "EnableMessageAggregation": "Off" 
```
The optimization should be enabled by explicit user request, e.g. when the workload contains a lot of small messages